src/server/page_server.cpp

```diff
@@ -224,10 +224,14 @@ page_server::tran_server_connection_handler::receive_start_catch_up (tran_server
   _er_log_debug (ARG_FILE_LINE,
 		 "[CATCH_UP] It's been requested to start catch-up with the follower (%s:%d), until LSA = (%lld|%d)\n",
 		 host.c_str (), port,LSA_AS_ARGS (&catchup_lsa));
+
   if (port == -1)
     {
       // TODO: It means that the ATS is booting up.
       // it will be set properly after ATS recovery is implemented.
+
+      // TODO reply the catchup_done msg.
```

Comment:
It appears that ATS is still booting, and there seems to be no information available for the main connection. In this situation, it doesn't seem advisable to request the prior node since catch-up hasn't finished. It appears that a catch-up retry is necessary. What do you think ?

Oh.. it can be addressed when ATS recovery is covered.